src/main/java/com/biel/lobby/mapes/jocs/OneInTheChamber.java

```diff
@@ -68,22 +73,27 @@ private Location getRandomSpawnLoc() {
 		items.add(p1.toItemStack(1));
 		return items;
 	}
-	protected void onPlayerDeathByPlayer(PlayerDeathEvent evt, Player killed,
-			Player killer) {
+
+	protected void onPlayerDeathByPlayer(PlayerDeathEvent evt, Player killed, Player killer) {
+
 		super.onPlayerDeathByPlayer(evt, killed, killer);
 		incrementScore(killer);
 		ItemStack Arrow = new ItemStack(Material.ARROW);
-		getStartingItems(killed);
 		killer.getInventory().addItem(Arrow);
 	}
-	protected void onPlayerDamageByPlayer(EntityDamageByEntityEvent evt, Player damaged, Player damager,
-			boolean ranged){
+
+	// Instant kill potion I guess
+	protected void onPlayerDamageByPlayer(EntityDamageByEntityEvent evt, Player damaged, Player damager, boolean ranged) {
+
 		super.onPlayerDamageByPlayer(evt, damaged, damager, ranged);
-		if(ranged)damaged.damage(1000);
+		if (ranged) evt.setDamage(100);
+
 	}
+
 	@Override
 	protected void onPlayerRespawnAfterTick(PlayerRespawnEvent evt, Player p) {
 		super.onPlayerRespawnAfterTick(evt, p);
 		teleportToRandomSpawn(p);
+		getStartingItems(p);
```

Comment:
Això la API té una manera de fer-ho. S'ha d'establir el giveStartingItemsOnRespawn a true (o similar)

Reply:
De fet crec que això no fa cap falta

Reply:
Val doncs si podeu treure això, estaria ready to merge

Reply:
Allow edits from maintainers esta activat, ho hauries de poder fer tu en un moment